lib/router/router.js

```diff
@@ -109,6 +126,17 @@ p.getInfo = function(req) {
     }
   }
 
+  // forward original image if no operation on image and browser support image format
+  if (routeInfo.imageSteps.length === 0 && (this.options.noop || routeInfo.urlInfo.query.noop)) {
```

Comment:
No. It seems that use original just avoid the creation of optimizedOriginal but it still process the cached version.
useOriginal can be set as a default using:   router: { originalSteps: false }
I was the first option I setup in my application other way I could not get the original file in full size.